src/main/src/test/java/org/geoserver/config/GeoServerImplTest.java

```diff
@@ -44,6 +44,21 @@ public void testGlobal() throws Exception {
         geoServer.setGlobal( global );
        
         assertEquals( global, geoServer.getGlobal() );
+
+        //GEOS-7890
+        GeoServerInfo global1 = geoServer.getFactory().createGlobal();
+        GeoServerInfo global2 = geoServer.getFactory().createGlobal();
+        global1.setGlobalServices(new Boolean(true));
+        global1.setXmlExternalEntitiesEnabled(new Boolean(false));
```

Comment:
Oh dear it is a Boolean boolean, and you needed to do this XML stuff to get two Boolean.TRUE values in the system?